### DISCUSSION TITLE
Supabase api error Bucket not found

### DISCUSSION THREAD

**Discussion by SaidovAliakbar** [supabase/supabase#30118] [Questions] [answered]
https://github.com/user-attachments/assets/fa561fe9-ce75-40eb-bc90-6a776ec9998a

StorageApiError: Bucket not found

**Comment by GaryAustin1** [comment, marked as answer]
You need to show your update and select policies not just the labels.   Likely you don't meet the policies.

**Reply by SaidovAliakbar** [reply]
The problem was on policies. I already fixed it. Thank you for wanting to help.
